libclingo/src/gringo_app.cc

```diff
@@ -163,6 +163,17 @@ struct IncrementalControl : Control, private Output::ASPIFOutBackend {
         parser.pushFile(std::string(filename), logger_);
         parse();
     }
+    void load_aspif(Potassco::Span<char const *> files) override {
+        for (auto it = end(files), ib = begin(files); it != ib; --it) {
```

Comment:
Nitpick: this (mostly) duplicated code is inconsistent regarding the use of begin/end. At least, I'd suggest dropping the (unnecessary) using declarations from the previous version.

Reply:
It does not compile without the using declarations. I admit, I did not check why.

Reply:
(The duplication will stay there for clingo series 5 but it will not stay like this in the upcoming 6.)